### PR TITLE
refine CUDA CPU places en doc

### PR DESCRIPTION
**Pull request by songyouwei**
Corresponding zh doc fix PR https://github.com/PaddlePaddle/FluidDoc/pull/1269

localhost_8000_documentation_docs_en_develop_api_fluid_cuda_places html (5)

localhost_8000_documentation_docs_en_develop_api_fluid_cuda_places html (4)

localhost_8000_documentation_docs_en_develop_api_fluid_cuda_places html (3)

localhost_8000_documentation_docs_en_develop_api_fluid_cuda_places html (2)

localhost_8000_documentation_docs_en_develop_api_fluid_cuda_places html (1)

localhost_8000_documentation_docs_en_develop_api_fluid_cuda_places html (6)

### CODE REVIEW

File: paddle/fluid/pybind/pybind.cc [hunk 1]
```diff
@@ -832,9 +832,23 @@ All parameter, weight, gradient are variables in Paddle.
...
+    staring from 0.
     The memory of CUDAPlace with different dev_id is not accessible.
+    Numbering here refers to the logical ID of the visible graphics card, not the actual ID of the graphics card.
+    You can set visible GPU devices by setting the `CUDA_VISIBLE_DEVICES` environment variable.
+    When the program starts, visible GPU devices will be numbered from 0.
+    If `CUDA_VISIBLE_DEVICES` is not set, all devices are visible by default,
+    and the logical ID is the same as the actual ID.
+
+    Parameters:
+        id (int, optional): GPU device ID. If it is None, the device with ID 0 will be used by default. Default: None.
```
Comment (chengduoZH) [inline paddle/fluid/pybind/pybind.cc]: The id is not optional here, and it must be set.

**Review by XiaoguangHu01 — DISMISSED** [review]

File: paddle/fluid/pybind/pybind.cc [hunk 2]
```diff
@@ -892,14 +906,14 @@ All parameter, weight, gradient are variables in Paddle.
       .def("__str__", string::to_string<const platform::CUDAPlace &>);
 
   py::class_<paddle::platform::CPUPlace>(m, "CPUPlace", R"DOC(
-    CPUPlace is a descriptor of a device. It represents a CPU, and the memory
-    CPUPlace can be accessed by CPU.
+    CPUPlace is a descriptor of a device.
+    It represents a CPU device allocated or to be allocated with Tensor or LoDTensor.
```
Comment (chengduoZH) [inline paddle/fluid/pybind/pybind.cc:910]: >> It represents a CPU device allocated or to be allocated with Tensor or LoDTensor.

What does this mean?

Comment (songyouwei) [reply paddle/fluid/pybind/pybind.cc:910]: 对应中文内容：

> CPUPlace 是一个设备描述符，表示一个分配或将要分配 Tensor 或 LoDTensor 的 CPU 设备。

**Review by wanghaoshuang — DISMISSED** [review]

File: paddle/fluid/pybind/pybind.cc (hunk 1 above)
Comment (songyouwei) [reply paddle/fluid/pybind/pybind.cc]: done

**Review by wanghaoshuang — DISMISSED** [review]

**Review by XiaoguangHu01 — DISMISSED** [review]

**Review by wanghaoshuang — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]

**Pull request merged**